core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -160,13 +160,15 @@ class KafkaApisTest extends Logging {
   }
 
   def createKafkaApis(
+    newGroupCoordiantorEnable: Boolean = true,
```

Comment:
That seems like very significant change. It shouldn't be necessary to enable the new group coordinator here. What are you trying to fix with this?